tests/spread/core24/linters-file/task.yaml

```diff
@@ -1,13 +1,17 @@
 summary: Run snapcraft lint on a snap file.
 
+environment:
+  SNAP: lint-file
+
 restore: |
-  rm -f ./*.snap ./*.assert
+  cd "${SNAP}"
+  rm -f ./*.snap ./*.assert ./linter_output.txt
+  rm -f ./*.snap
```

Comment:
```suggestion
  rm -f ./*.snap ./*.assert ./linter_output.txt
```

Reply:
I will do this when I rebase the PR you took over